packages/renderer/src/lib/ui/EngineFormPage.svelte

```diff
@@ -8,10 +8,14 @@ export let showEmptyScreen: boolean = false;
 </script>
 
 <FormPage title={title} inProgress={inProgress}>
-  <slot slot="icon" name="icon" />
-  <slot slot="actions" name="actions" />
-  <slot slot="tabs" name="tabs" />
-  <slot slot="content">
+  {#snippet icon()}
+  <slot name="icon" />
```

Comment:
question: is it necessary to keep the slot in it? like is this for backward compatibility?

Reply:
This slot is necessary, as this component is still in svelte 4.

When migrated to svelte 5, this part will be:

```
{#snippet icon()}{@render localIcon?.()}{/snippet}
```